models/bidDao.js

```diff
@@ -0,0 +1,44 @@
+const appDataSource = require("./dataSource")
+
+const getBidEventsToken = async (eventId) => {
+  const oneWeekAgo = new Date();
+  oneWeekAgo.setDate(oneWeekAgo.getDate() - 7);
+
+  try {
+    const graph = await appDataSource.query(
+      `
+      SELECT
+        event_id,
+        bidding_events_token,
+        created_at
```

Comment:
SQL을 다음과 같이 가져오면 아래 로직들은 불필요해 보입니다.
```suggestion
        JSON_OBJECT(
          "y" , bidding_events_token,
          "x", created_at
        ) AS data
```